avro-fastserde/src/main/java/com/linkedin/avro/fastserde/SchemaAssistant.java

```diff
@@ -63,7 +63,7 @@
   public SchemaAssistant(JCodeModel codeModel, boolean useGenericTypes, Class defaultStringClass) {
     this.codeModel = codeModel;
     this.useGenericTypes = useGenericTypes;
-    this.exceptionsFromStringable = new HashSet<>();
+    this.exceptionsFromStringable = new TreeSet<>(Comparator.comparing(Class::getCanonicalName));
```

Comment:
curious to know why changing to use `TreeSet` here?

Reply:
I should add some comments here, and this is used to make the code gen deterministic in FastDeserializerGenerator.java:
```
for (Class<? extends Exception> classException : schemaAssistant.getExceptionsFromStringable()) {
          JCatchBlock catchBlock = tryBlock._catch(codeModel.ref(classException));
          JVar exceptionVar = catchBlock.param("e");
          catchBlock.body()._throw(JExpr._new(codeModel.ref(AvroRuntimeException.class)).arg(exceptionVar));
        }
```